app/models/course.rb

```diff
@@ -1,27 +1,25 @@
 class Course < ActiveRecord::Base
-    has_many :enrollments, dependent: :delete_all
-    has_many :users, through: :enrollments
+  has_many :enrollments, dependent: :delete_all
+  has_many :users, through: :enrollments
 
-    has_many :teacher_enrollments, -> {Enrollment.teacher}, class_name: 'Enrollment'
-    has_many :teachers, through: :teacher_enrollments, class_name: 'User', source: 'user'
+  has_many :teacher_enrollments, -> {Enrollment.teacher}, class_name: 'Enrollment'
```

Comment:
Line is too long. [82/80]<br>Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.